docs/guides/jobs/metrics.md

```diff
@@ -0,0 +1,45 @@
+FloydHub provides various metrics for your training jobs in order to help you measure how well your job's training process is going. These metrics are visible as graphs on the job overview page.
+
+Metrics are collected once every 60 seconds and will be updated in real time during a job's training process. Metrics will be retained after the job is completed - however, we are still determining the default retention period.
+
+Metrics are grouped into two categories:
+
+* **System Metrics**: System stats, such as CPU or GPU utilization
+* **Training Metrics**: Custom training metrics, like training accuracy, loss, etc.
+
+## System Metrics
+
+System metrics provide insight into the system stats of your FloydHub training machine. These metrics will be automatically collected for all types of jobs - Command Mode, Jupyter Mode, and Serving Mode.
+
+For all jobs, you will receive:
+
+* **CPU Utilization:** Percentage of CPU usage by your training job
+* **Memory Utilization:** Percentage of RAM usage by your training job
+* **Disk Utilization:** Percentage of SSD usage by your training job
+
+Additionally, for any GPU-powered jobs, you will receive:
+
+* **GPU Utilization:** Percentage of GPU usage by your training job
+* **GPU Memory Utilization:** Percentage GPU Memory by your training job
+
```

Comment:
- Add a screenshot
- Explain how this can be useful. E.g. "the GPU memory utilization graph can help you optimize your job. You can increase or decrease your batch size to ensure you're fully utilizing your GPU. It can also help you debug failed jobs due to out-of-memory (OOM) errors"